inc/functions/files.php

```diff
@@ -1470,3 +1470,32 @@ function _rocket_is_windows_fs( $hard_reset = false ) { // phpcs:ignore WordPres
 function _rocket_get_wp_rocket_cache_path() { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedFunctionFound
 	return _rocket_normalize_path( rocket_get_constant( 'WP_ROCKET_CACHE_PATH' ) );
 }
+
+/**
+ * Get an array of php files in directory.
+ *
+ * @param string $dir_path Full path to the directory.
+ *
+ * @return array php files in the directory.
+ *               [...SplFileInfo file]
+ *
+ * @since  ver 3.6.1
+ *
+ * @author Caspar Green
+ */
+function _rocket_get_directory_php_files_array( $dir_path ) { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedFunctionFound
```

Comment:
I'm wondering if we can shorten this function name while making it focus on its expected behavior. Thinking out loud, this function does the following:

- Gets all of the `.php` files in the given directory.
- It's not recursive. It only gets them in this given directory and not the subdirectories.

What do you think of this name:

`_rocket_get_php_files_in_dir()`

Reply:
Kinda thinking that if @engahmeds3ed 's regex approach, above, gets merged, this whole thing is a duplication.

Reply:
Renamed.